nibabies/workflows/bold/resampling.py

```diff
@@ -135,48 +199,422 @@ def select_target(subject_id, space):
+    )
+
+    create_pial_distvol = pe.MapNode(
+        CreateSignedDistanceVolume(),
+        iterfield=["surface"],
+        name="create_pial_distvol",
+        mem_gb=mem_gb,
+    )
+
+    thresh_wm_distvol = pe.MapNode(
+        fsl.maths.MathsCommand(args="-thr 0 -bin -mul 255"),
+        iterfield=["in_file"],
+        name="thresh_wm_distvol",
+        mem_gb=DEFAULT_MEMORY_MIN_GB,
+    )
+
+    uthresh_pial_distvol = pe.MapNode(
+        fsl.maths.MathsCommand(args="-uthr 0 -abs -bin -mul 255"),
+        iterfield=["in_file"],
+        name="uthresh_pial_distvol",
+        mem_gb=DEFAULT_MEMORY_MIN_GB,
+    )
+
+    bin_wm_distvol = pe.MapNode(
+        fsl.maths.UnaryMaths(operation="bin"),
+        iterfield=["in_file"],
+        name="bin_wm_distvol",
+        mem_gb=DEFAULT_MEMORY_MIN_GB,
+    )
+
+    bin_pial_distvol = pe.MapNode(
+        fsl.maths.UnaryMaths(operation="bin"),
+        iterfield=["in_file"],
+        name="bin_pial_distvol",
+        mem_gb=DEFAULT_MEMORY_MIN_GB,
+    )
+
+    split_wm_distvol = pe.Node(
+        niu.Split(splits=[1, 1]),
+        name="split_wm_distvol",
+        mem_gb=DEFAULT_MEMORY_MIN_GB,
+    )
+
+    merge_wm_distvol_no_flatten = pe.Node(
+        niu.Merge(2),
+        no_flatten=True,
+        name="merge_wm_distvol_no_flatten",
+        mem_gb=DEFAULT_MEMORY_MIN_GB,
+    )
+
+    make_ribbon_vol = pe.MapNode(
+        fsl.maths.MultiImageMaths(op_string="-mas %s -mul 255 "),
+        iterfield=["in_file", "operand_files"],
+        name="make_ribbon_vol",
+        mem_gb=DEFAULT_MEMORY_MIN_GB,
+    )
+
+    bin_ribbon_vol = pe.MapNode(
+        fsl.maths.UnaryMaths(operation="bin"),
+        iterfield=["in_file"],
+        name="bin_ribbon_vol",
+        mem_gb=DEFAULT_MEMORY_MIN_GB,
+    )
+
+    split_squeeze_ribbon_vol = pe.Node(
+        niu.Split(splits=[1, 1], squeeze=True),
+        name="split_squeeze_ribbon",
+        mem_gb=DEFAULT_MEMORY_MIN_GB,
+    )
+
+    combine_ribbon_vol_hemis = pe.Node(
+        fsl.maths.BinaryMaths(operation="add"),
+        name="combine_ribbon_vol_hemis",
+        mem_gb=DEFAULT_MEMORY_MIN_GB,
+    )
+
+    ribbon_boldsrc_xfm = pe.Node(
+        ApplyTransforms(interpolation='MultiLabel',
+                        transforms='identity'),
+        name="ribbon_boldsrc_xfm",
+        mem_gb=mem_gb,
+    )
+
+    stdev_volume = pe.Node(
+        fsl.maths.StdImage(dimension='T'),
+        name="stdev_volume",
+        mem_gb=DEFAULT_MEMORY_MIN_GB,
+    )
+
+    mean_volume = pe.Node(
+        fsl.maths.MeanImage(dimension='T'),
+        name="mean_volume",
+        mem_gb=DEFAULT_MEMORY_MIN_GB,
+    )
+
+    cov_volume = pe.Node(
+        fsl.maths.BinaryMaths(operation='div'),
+        name="cov_volume",
+        mem_gb=DEFAULT_MEMORY_MIN_GB,
+    )
+
+    cov_ribbon = pe.Node(
+        fsl.ApplyMask(),
+        name="cov_ribbon",
+        mem_gb=DEFAULT_MEMORY_MIN_GB,
+    )
+
+    cov_ribbon_mean = pe.Node(
+        fsl.ImageStats(op_string='-M '),
```

Comment:
Trailing spaces again.

```suggestion
        fsl.ImageStats(op_string='-M'),
```